project/dexter-monitor/public/service/userService.js

```diff
@@ -27,30 +27,27 @@
 
 monitorApp.service('UserService', function($http, $log, $q) {
 
-    this.getUserListByProject = function(projectName) {
-        if (!projectName)
-            return $q.reject('Project name is null');
+    this.getExtraInfoByUserId = function(userId) {
+        if (!userId || userId.length == 0)
+            return $q.reject('User ID is invalid');
 
-        return $http.get('/api/v2/user/project/' + projectName)
+        return $http.get('/api/v2/user/extra-info/' + userId)
             .then((res) => {
                 if (!isHttpResultOK(res)) {
-                    $log.error('Failed to get user list');
-                    return [];
+                    $log.error('Failed to load extra user info');
+                    return null;
                 }
 
-                return res.data.rows;
+                return res.data.rows[0];
             })
             .catch((err) => {
                 $log.error(err);
```

Comment:
Added alert dialog in `getExtraInfo` function of `currentUserCtrl.js`
`Failed to load extra information of '${user.userId}'.`
`Please contact the system administrator.`